homework-3/src/views/SignUp.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { StylBoxMain } from '../components/common/Boxes/StylBoxMain';
+import FormSignUp from '../components/Forms/FormSignUp';
+import BoxForm from './../components/common/Boxes/StylBoxForm';
+import BoxContentLink from './../components/Boxes/BoxContentLink';
+import Loader from './../components/Loader/Loader';
+
+const SignUp = ({ title }) => {
+  React.useEffect(() => {
+    document.title = title || '';
+  }, [title]);
+
+  const [loading, setLoading] = React.useState(true);
+
+  React.useEffect(() => {
+    const loadData = async () => {
+      await new Promise((r) => setTimeout(r, 2000));
+
+      setLoading((loading) => !loading);
+    };
+
+    loadData();
+  }, []);
```

Comment:
What is loading here didnt get it? what type of data is loading now?

Reply:
loadData was a promise, which is a kind of data. That's what he called it.
Now, useEffect with loader removed